pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,23 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+
+- List the current settings:
+
+`efibootmgr -v for Filepaths`
+
+- Create a new boot option:
+
+`sudo efibootmgr -c -d {{esp_blockdevice}} -l {{\path\to\file.efi}} -L "{{Label}}"`
```

Comment:
The convention for path tokens we follow is "path/to/file.efi". And please give a concrete example for block device - like "{{/dev/sda1}}" and same for label.

Reply:
The backslashes are there, because the ESP is a FAT Filesystem, and "officially" FAT uses \. `efibootmgr -v` is displaying the Paths using \, the Manpage is using \. And UEFI Shell also displays \.
We have had the same thing somewhere in this PR already.

Reply:
The `/` convention is something we decided to use irrespective of what platform the page is for.

Reply:
I think I remember pointing this one out before @agnivade before - in this case it's not a regular file path. It's an EFI file path, which _must_ use backward slashes - otherwise it doesn't work IIRC.

So in this case I think we might be able to make an exception.